examples/core_api/timm-post/post.md

```diff
@@ -0,0 +1,286 @@
+(This post references unreleased features, namely https://github.com/determined-ai/determined/pull/3859 and https://github.com/determined-ai/determined/pull/3807.  To run examples, you'll need to create a local branch with those two PRs merged.)
+
+(TODO: Replace github references with doc references where possible.)
+
+(TODO: Are the initial paragraphs the right story to tell about Core API?  Could completely replace them.)
```

Comment:
Agreed, we should tell a better story for marketing reasons.  I'll write something up here shortly.